tools/install/terrascan.sh

```diff
@@ -0,0 +1,29 @@
+#!/usr/bin/env bash
+set -eo pipefail
+# shellcheck disable=SC1091 # Created by Dockerfile above script call
+source /.env
+
+if [[ $TERRASCAN_VERSION != false ]]; then
+  if [[ $TARGETARCH != amd64 ]]; then
+    ARCH="$TARGETARCH"
+  else
+    ARCH="x86_64"
+  fi
+  # Convert the first letter to Uppercase
```

Comment:
This for sure can be converted into simpler version:
```suggestion
  [[ $TARGETARCH != amd64 ]] && ARCH="$TARGETARCH" || ARCH="x86_64"

  # Convert the first letter to Uppercase
```